Content.Server/Chemistry/EntitySystems/SolutionContainerSystem.Capabilities.cs

```diff
@@ -113,6 +113,14 @@ public FixedPoint2 DrainAvailable(EntityUid uid)
             : solution.CurrentVolume;
     }
 
+    public int PercentFull(EntityUid uid)
+    {
+        if (!TryGetDrainableSolution(uid, out var solution) || solution.MaxVolume.Equals(FixedPoint2.Zero))
+            return 0;
+
+        return (int)((solution.CurrentVolume.Float() / solution.MaxVolume.Float()) * 100);
+    }
+
```

Comment:
i dont think theres any reason to not just make this return a float, and do the casting in whatever method calls it rather than losing information